config/config.go

```diff
@@ -82,7 +83,9 @@ func init() {
 type Server struct {
 	BindHost    string
 	BindPort    int
+	Scheme      string
```

Comment:
Let's prepend these three values with something ("serve_" or "url_") to make it clear they go together for building urls, not that they actually change where stuff is served.